tests/tests_data/test_mri_transforms.py

```diff
@@ -262,6 +266,41 @@ def test_CropKspace(
         assert sample["kspace"].shape == (shape[0],) + crop_shape + (2,)
 
 
+@pytest.mark.parametrize(
+    "shape, pad_shape, mode",
+    [
+        [(3, 10, 16), (20, 26), RescaleMode.NEAREST],
+        [(3, 10, 16), (20, 26), RescaleMode.AREA],
+        [(3, 10, 16), (20, 26), RescaleMode.BILINEAR],
+        [(3, 10, 16), (20, 26), RescaleMode.BICUBIC],
+        [(3, 21, 10, 16), (30, 20, 26), RescaleMode.NEAREST],
+        [(3, 21, 10, 16), (30, 20, 26), RescaleMode.AREA],
+        [(3, 21, 10, 16), (30, 20, 26), RescaleMode.TRILINEAR],
+    ],
+)
+def test_RescaleKspace(shape, pad_shape, mode):
+    sample = create_sample(shape=shape + (2,))
+    transform = RescaleKspace(pad_shape, rescale_mode=mode)
+
+    sample = transform(sample)
+    assert sample["kspace"].shape == shape[: -len(pad_shape)] + pad_shape + (2,)
```

Comment:
Don't you want to test if the interpolation is done well?

Reply:
I am using pytorch's interpolation so I cannot really check that unless I compare to pytorch's interpolation, (if it makes sense what I am saying )